docs/file-formats/json-project/overview.md

```diff
@@ -216,6 +216,9 @@ Defines the permissions that will be applied to new resources/values.
 - `public`: All users can view resources/values.
 - `private`: Only ProjectMembers/ProjectAdmins can view resources/values.
 
+If you work with the Excel templates,
+this setting is available in the [`json_header.xlsx` file](../excel2json.md#the-json_headerxlsx).
+
```

Comment:
That's how it looks like: 

<img width="1594" height="954" alt="Screenshot 2025-08-25 at 15 16 03" src="https://github.com/user-attachments/assets/6a925f77-5236-43a7-9853-67f58626c0ad" />